python/lsst/ts/electrometer/csc.py

```diff
@@ -154,18 +209,24 @@ async def do_setDigitalFilter(self, data):
             substates=[DetailedState.NOTREADINGSTATE],
             action="setDigitalFilter",
         )
-        await self.report_detailed_state(DetailedState.CONFIGURINGSTATE)
-        await self.controller.set_digital_filter(
-            activate_filter=data.activateFilter,
-            activate_avg_filter=data.activateAvgFilter,
-            activate_med_filter=data.activateMedFilter,
-        )
-        self.log.debug("setDigitalFilter controller interaction completed")
-        self.log.debug(f"filter_active={self.controller.filter_active}")
-        self.log.debug(f"avg_filter_active={self.controller.avg_filter_active}")
-        self.log.debug(f"median_filter_active={self.controller.median_filter_active}")
-        await self.report_detailed_state(DetailedState.NOTREADINGSTATE)
-        self.log.info("setDigitalFilter Completed")
+        try:
+            await self.report_detailed_state(DetailedState.CONFIGURINGSTATE)
+            await self.controller.set_digital_filter(
+                activate_filter=data.activateFilter,
+                activate_avg_filter=data.activateAvgFilter,
+                activate_med_filter=data.activateMedFilter,
+            )
+            self.log.debug("setDigitalFilter controller interaction completed")
+            self.log.debug(f"filter_active={self.controller.filter_active}")
+            self.log.debug(f"avg_filter_active={self.controller.avg_filter_active}")
+            self.log.debug(
+                f"median_filter_active={self.controller.median_filter_active}"
+            )
```

Comment:
Please consider combining the above three log messages (filter_active through median_filter_active) since they all seem to be related.